apps/etl/models.py

```diff
@@ -0,0 +1,95 @@
+# Create your models here.
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from apps.common.models import UserResource
+
+
+class HazardType(models.TextChoices):
+    EARTHQUAKE = "EQ", "Earthquake"
+    FLOOD = "FL", "Flood"
+    CYCLONE = "TC", "Cyclone"
+    EPIDEMIC = "EP", "Epidemic"
+    FOOD_INSECURITY = "FI", "Food Insecurity"
+    STORM = "SS", "Storm Surge"
+    DROUGHT = "DR", "Drought"
+    TSUNAMI = "TS", "Tsunami"
+    WIND = "CD", "Cyclonic Wind"
+    WILDFIRE = "WF", "WildFire"
+    VOLCANO = "VO", "Volcano"
+
+
+class ExtractionData(UserResource):
+    class ValidationStatus(models.IntegerChoices):
+        SUCCESS = 1, _("Success")
+        FAILED = 2, _("Failed")
+        NO_DATA = 3, _("No data")
+        NO_CHANGE = 4, _("No change")
+        NO_VALIDATION = 5, _("No validation")
+
+    class ResponseDataType(models.IntegerChoices):
+        JSON = 1, _("json")
+        CSV = 2, _("csv")
+        TEXT = 3, _("text")
+        HTML = 4, _("html")
+        XML = 5, _("xml")
+        PDF = 6, _("pdf")
+
+    class Source(models.IntegerChoices):
+        GDACS = 1, _("GDACS")
+        PDC = 2, _("PDC")
+
+    class Status(models.IntegerChoices):
+        PENDING = 1, _("Pending")
+        IN_PROGRESS = 2, _("In progress")
+        SUCCESS = 3, _("Success")
+        FAILED = 4, _("Failed")
+
+    source = models.IntegerField(verbose_name=_("source"), choices=Source.choices)
+    url = models.URLField(verbose_name=_("url"), blank=True)
+    attempt_no = models.IntegerField(verbose_name=_("attempt number"), blank=True)
+    resp_code = models.IntegerField(verbose_name=_("response code"), blank=True)
+    status = models.IntegerField(verbose_name=_("status"), choices=Status.choices)
+    resp_data = models.FileField(verbose_name=_("response data"), upload_to="source_raw_data/", blank=True, null=True)
+    file_hash = models.CharField(
```

Comment:
Add TODO: change to resp_data -> response_file

```suggestion
    response_file = models.FileField(verbose_name=_("response data"), upload_to="source_raw_data/", blank=True, null=True)
    response_file_hash = models.CharField(
```